README.md

```diff
@@ -102,6 +102,44 @@ pdk test unit
 
 This command runs all available unit tests.
 
+### Convert an existing module
+
+Aside from creating new modules, the PDK now has the ability to convert an existing module into a PDK ?enabled? module. You can then manage and test your module using the tools bundled with the PDK.
```

Comment:
Let's say "PDK **managed** module" for now. Makes it clear that the PDK has an active interest in the module's contents/setup. 

cc @rickmonro